tests/foreman/api/test_provisioningtemplate.py

```diff
@@ -649,14 +659,18 @@ def test_positive_template_check_rex_pull_mode_snippet(
         default_partitiontable,
     ):
         """Read the provision template and verify the host params and REX pull mode snippet rendered correctly.
+           Verify chronyd is getting installed.
```

Comment:
Again, same for this, why we checking chronyd or ntpdate package installs in rex_pull_mode_snippet test?

Reply:
Removed from this test and added a new test.